### PR TITLE
added showcase route, jade and  images

### PR DESCRIPTION
**Pull request by jmervine**
Create a PR to discuss.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -25,13 +25,18 @@ function legacy(req, res) {
...
+}
+
 module.exports = {
-    index:         index,
-    fontawesome:   fontawesome,
-    bootswatch:    bootswatch,
-    bootlint:      bootlint,
-    alpha:         alpha,
-    legacy:        legacy
+  index:         index,
```
Comment (jmervine) [inline routes/index.js]: JS lint is 4 space indent. :-P

File: views/_partials/header.jade [hunk 2]
```diff
@@ -10,13 +10,14 @@
             a.navbar-brand(href="/" + themeQs) Bootstrap CDN
         .navbar-collapse.collapse
             ul.nav.navbar-nav
-                li: a(href="/fontawesome/" + themeQs): i.fa.fa-flag 
+                li: a(href="/fontawesome/" + themeQs): i.fa.fa-flag
```
Comment (jmervine) [inline views/_partials/header.jade]: Add a space on the end of this line, otherwise formatting is going to be off.

File: views/showcase.jade [hunk 3]
```diff
@@ -0,0 +1,18 @@
+extends layout
+
+block content
+    h2 Showcase
+    - each item in config.showcase
+        - if (!item.latest)
+            div.col-md-6.text-center
```
Comment (jmervine) [inline views/showcase.jade]: This doesn't need the `div`, it's automatically added and should have an offset, i.e. `.col-md-6.col-md-offset-3.text-center`.

File: views/showcase.jade [hunk 4]
```diff
@@ -0,0 +1,18 @@
+extends layout
+
+block content
+    h2 Showcase
+    - each item in config.showcase
```
Comment (jmervine) [inline views/showcase.jade]: To match style/formatting of the reset of the site, this should match bootswatch (see: https://github.com/MaxCDN/bootstrap-cdn/blob/develop/views/bootswatch.jade). You need wrap everything after the `h2 Showcase` in a `.well.text-center`, but can replace the `row` with the `offset` element I mentioned below.

File: config/_config.yml [hunk 5]
```diff
@@ -250,3 +250,40 @@ fontawesome:
   - version: 3.2.1
     stylesheet: 'https://maxcdn.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.min.css'
     stylesheet_sri: sha256-MmuZTsWcczT1IhH71aqQmja5jRcXy3mL/NOvjUy9tso= sha512-IJ+BZHGlT4K43sqBGUzJ90pcxfkREDVZPZxeexRigVL8rzdw/gyJIflDahMdNzBww4k0WxpyaWpC2PLQUWmMUQ==
+showcase:
```
Comment (jmervine) [inline config/_config.yml]: Feel free to ignore this, just a suggestion, but the key's feel a bit verbose here. I would use `name`, `img`, `lib`.

File: views/_partials/header.jade [hunk 6]
```diff
@@ -10,13 +10,14 @@
...
                     | Font Awesome
                 li: a(href="/bootswatch/" + themeQs) Bootswatch
                 li: a(href="/bootlint/" + themeQs) Bootlint
                 li: a(href="/legacy/" + themeQs) Legacy
                 li: a(href="/alpha/" + themeQs) Bootstrap4
                     sup ALPHA
+                li: a(href="/showcase/" + themeQs) Showcase
```
Comment (jmervine) [inline views/_partials/header.jade]: Your call, but I would keep BS4 Alpha last.

Comment (jdorfman) [inline views/_partials/header.jade]: yeah that is fine

**Pull request merged**